account/views.py

```diff
@@ -102,4 +102,29 @@ def post(self, request):
         new_access_token = str(RefreshToken(refresh_token).access_token)
         response = Response({"detail": "token refreshed"}, status=status.HTTP_200_OK)
         response.set_cookie("access_token", value=str(new_access_token), httponly=True)
-        return response
\ No newline at end of file
+        return response
+    
+class LogoutView(APIView):
+    @swagger_auto_schema(
+        operation_id="로그아웃",
+        operation_description="refresh token을 삭제합니다.",
```

Comment:
'사용자를 로그아웃시킵니다'가 기능을 조금 더 직관적으로 설명하는 것 같아서 추천드립니다 ! (swagger는 백엔드 개발에서 api 테스트를 할 때도 사용되지만 다른 팀과의 소통을 위해서도 사용되기 때문에 최대한 직관적으로 설명을 달아주는 게 좋아용)
```suggestion
        operation_description="사용자를 로그아웃시킵니다.",
```

Reply:
참고하겠습니다!